content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: blog
+title: "From Code to Quill: Embark on a Legendary Kubernetes Quest with SIG Docs"
+date: 2024-05-16
+slug: contribute-as-sig-docs-reviewer
```

Comment:
As this is a [mirrored](https://github.com/kubernetes/contributor-site/pull/496) blog article, we should consider including `canonicalUrl` in the front matter.

```suggestion
slug: contribute-as-sig-docs-reviewer
canonicalUrl: https://www.kubernetes.dev/blog/2024/05/16/contribute-as-sig-docs-reviewer/
```